app/controllers/concerns/mfa_setup_concern.rb

```diff
@@ -6,17 +6,18 @@ def next_setup_path
       auth_method_confirmation_url
     elsif next_setup_choice
       confirmation_path
-    else
-      if user_session[:mfa_selections]
-        analytics.user_registration_mfa_setup_complete(
-          mfa_method_counts: mfa_context.enabled_two_factor_configuration_counts_hash,
-          enabled_mfa_methods_count: mfa_context.enabled_mfa_methods_count,
-          pii_like_keypaths: [[:mfa_method_counts, :phone]],
-          success: true,
-        )
-      end
-      user_session.delete(:mfa_selections)
-      nil
+    elsif user_session[:mfa_selections]
+      analytics.user_registration_mfa_setup_complete(
+        mfa_method_counts: mfa_context.enabled_two_factor_configuration_counts_hash,
+        enabled_mfa_methods_count: mfa_context.enabled_mfa_methods_count,
+        pii_like_keypaths: [[:mfa_method_counts, :phone]],
+        success: true,
+      )
+      second_mfa_setup_url
+      else
+        user_session.delete(:mfa_selections)
+        nil
+
```

Comment:
Minor: Can we fix up the whitespace / indentation here?

```suggestion
    else
      user_session.delete(:mfa_selections)
      nil
```

Reply:
Fixed that.